README.md

```diff
@@ -128,6 +131,9 @@ For Windows:
 
 ## Rucio usage
 
+_NOTE: You can execute the Linux script for better set up and understanding:
```

Comment:
same as above

Reply:
Fixed in the commit 84bf9a925933eebb4b73482beb79825180033d7d.
Fixed in the commit 508e8d596246911768f13810b48d7085a1600ffb.